codegen/src/test/resources/software/amazon/awssdk/codegen/poet/eventstream/defaultsecondeventtwo.java

```diff
@@ -1,21 +1,23 @@
-package software.amazon.awssdk.services.jsonprotocoltests.model.eventstream;
+package software.amazon.awssdk.services.json.model.eventstream;
 
 import software.amazon.awssdk.annotations.Generated;
 import software.amazon.awssdk.annotations.SdkInternalApi;
-import software.amazon.awssdk.services.jsonprotocoltests.model.EventStreamOperationResponseHandler;
-import software.amazon.awssdk.services.jsonprotocoltests.model.EventTwo;
+import software.amazon.awssdk.services.json.model.EventStream;
+import software.amazon.awssdk.services.json.model.EventStreamOperationResponseHandler;
+import software.amazon.awssdk.services.json.model.EventStreamOperationWithOnlyOutputResponseHandler;
+import software.amazon.awssdk.services.json.model.EventTwo;
 
 /**
- * A specialization of {@code software.amazon.awssdk.services.jsonprotocoltests.model.EventTwo} that represents the
- * {@code EventStream#SecondEventTwo} event. Do not use this class directly. Instead, use the static builder methods on
- * {@link software.amazon.awssdk.services.jsonprotocoltests.model.EventStream}.
+ * A specialization of {@code software.amazon.awssdk.services.json.model.EventTwo} that represents the
+ * {@code EventStream$secondeventtwo} event. Do not use this class directly. Instead, use the static builder methods on
+ * {@link software.amazon.awssdk.services.json.model.EventStream}.
  */
 @SdkInternalApi
 @Generated("software.amazon.awssdk:codegen")
-public final class DefaultSecondEventTwo extends EventTwo {
+public final class DefaultSecondeventtwo extends EventTwo {
```

Comment:
Is this change intentional? Shouldn't it be `DefaultSecondEventTwo`?

Reply:
It's a little confusing, but this one corresponds to this event https://github.com/aws/aws-sdk-java-v2/blob/32cd8d61bc470ef325477fa29af90d5b8c12b2b4/codegen/src/test/resources/software/amazon/awssdk/codegen/poet/eventstream/service-2.json#L106